tests/scripts/create_coverage_report.py

```diff
@@ -5,7 +5,6 @@
 import pathlib
 
 from typing import Dict
-from bluechi_test import command
```

Comment:
Please do not remove, it's important for multinode tests

Reply:
Are we already working on the multi-node?
If so, then there should probably be a branch with this and it can safely be removed here. If not, its still unused code - I don't see the need for keeping it for a potential use in the future.

Reply:
We have command.py in theh project for long time and at minimum it's used for log gathering: https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/test.py#L148

Reply:
Ah ok, I missed that. In that case yes, lets keep it.